ddl/foreign_key_test.go

```diff
@@ -1426,6 +1424,26 @@ func TestAddForeignKey(t *testing.T) {
 	tk.MustExec("alter table t1 add index(b)")
 	tk.MustExec("alter table t2 add foreign key (b) references t1(b);")
 	tk.MustGetDBError("alter table t2 add foreign key (b) references t2(b);", infoschema.ErrCannotAddForeign)
+	// Test auto-create index when create foreign key constraint.
```

Comment:
Could you add a test for creating multiple foreign keys?

Reply:
nice catch , done.